sites/public/pages/applications/review/terms.tsx

```diff
@@ -117,7 +119,11 @@ export default () => {
           </div>
           <div className="form-card__pager">
             <div className="form-card__pager-row primary">
-              <Button styleType={AppearanceStyleType.primary} onClick={() => false}>
+              <Button
+                loading={submitting}
+                styleType={AppearanceStyleType.primary}
+                onClick={() => false}
```

Comment:
If this onClick isn't functionally doing anything we can remove it :) onClick is now optional - can't tell if the false is needed or if it was just a placeholder when this was a required field

Reply:
Ah yeah, must have been a holdover from before. I will remove!